doc/tutorials/enable_s5.rst

```diff
@@ -205,3 +205,45 @@ How to test
 
       # acrnctl list
       vm1		stopped
+
+System Shutdown
+***************
+
+Using a coordinating script, ``misc/life_mngr/s5_trigger.sh``, in conjunction with
+the lifecycle manager in each VM, graceful system shutdown can be performed.
+
+.. note:: Please install ``s5_trigger.sh`` manually to your home directory.
```

Comment:
Shouldn't that be the `root` user home directory instead?

Reply:
> Shouldn't that be the `root` user home directory instead?

That's a good suggestion. Since the user usually starts `life_mngr` via `life_mngr.service`, installing to `root`'s home directory makes sense. If a user manually starts `life_mngr` via `sudo`, then `life_mngr` will look for the script in that user's home directory. However, this seems more like a testing scenario.